content/blog/noetic-eol/index.md

```diff
@@ -0,0 +1,178 @@
+
+**Changes to the ROS API** 
+
+Most of the ROS 2 API should look familiar to ROS 1 users with a few notable exceptions. 
+Where things have changed, they have changed to address shortcomings in ROS 1. 
+The exciting thing about ROS 2 is that it has given us the opportunity to address a number of user complaints about ROS 1! 
+The list below outlines some of the larger changes to the ROS API found in version 2.
+
+* Nodelets in ROS 1 have been replaced with ROS 2's [composable nodes concept](https://docs.ros.org/en/rolling/Tutorials/Intermediate/Composition.html). ROS users can now compose their processing routines into the same process space using both compile time and run time composition. 
+* Parameters in ROS 2 now support namespacing, and there is an improved interface for loading, saving, and reconfiguring your parameters. This [migration guide](https://docs.ros.org/en/rolling/How-To-Guides/Migrating-from-ROS1/Migrating-Parameters.html) provides guidance on how to migrate your parameter set.
+* ROS 2 supports three different types of launch files: [YAML, Python, and XML](https://docs.ros.org/en/rolling/Tutorials/Intermediate/Launch/Creating-Launch-Files.html). There is a complete guide to [migrating your ROS 1 launch files](https://docs.ros.org/en/rolling/How-To-Guides/Migrating-from-ROS1/Migrating-Launch-Files.html) available on docs.ros.org.
+* The ROS 2 [lifecycle node API](https://foxglove.dev/blog/how-to-use-ros2-lifecycle-nodes) has standardized the lifecycle of ROS nodes. The process of starting and stopping ROS nodes is now much more robust. We recommend taking advantage of this feature when migrating your ROS 1 project. 
+* ROSBag performance has improved significantly in ROS 2, and the new ROS 2 type adaptation makes it even easier to share data using ROS Bags. As of ROS 2 Humble [MCap](https://mcap.dev/guides) is now the standard ROSBag data format.
+* If you are looking for examples of common ROS 2 design patterns the [ROS 2 Cookbook](https://github.com/mikeferguson/ros2_cookbook) offers code samples for common ROS 2 tasks.
+
+<br>
+
+**ROS Development**
+
+The ROS build and command line tools have also gotten a face lift in ROS 2.
+While there have been a few changes, the way you build and run your ROS application should feel familiar. All of our new tools are self-documenting, and you can learn a lot by simply passing them the `--help` option.
+
+* The [ROS command line interface (CLI)](https://docs.ros.org/en/rolling/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Services/Understanding-ROS2-Services.html) syntax has changed slightly in ROS 2. All your favorite commands are still there, but the syntax has been modernized.
+* ROS 2 uses [Colcon](https://colcon.readthedocs.io/en/released/user/quick-start.html), rather than `catkin-build`, as the build tool. The names have changed, but most of your favorite functionality is still there.
```

Comment:
As argued in https://github.com/ros-infrastructure/www.ros.org/pull/54#discussion_r1927664150, `catkin-build` is not a valid build tool at all. We have: `catkin_make`, `catkin_make_isolated`, and `catkin build` (without a hyphen).